### PR TITLE
Misc UI improvements

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

File: ashpd-demo/data/com.belmoussaoui.ashpd.demo.metainfo.xml.in.in [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-<!-- Bilal Elmoussaoui 2019 <bil.elmoussaoui@gmail.com> -->
+<!-- Bilal Elmoussaoui 2021 <bil.elmoussaoui@gmail.com> -->
 <component type="desktop-application">
   <id>@app-id@</id>
   <metadata_license>CC0</metadata_license>
-  <project_license>GPL-3.0+</project_license>
+  <project_license>MIT</project_license>
   <name>ASHPD Demo</name>
-  <summary>A GTK + Rust application template.</summary>
+  <summary>Play with portals.</summary>
```
Comment (bilelmoussaoui) [inline ashpd-demo/data/com.belmoussaoui.ashpd.demo.metainfo.xml.in.in:8]: No "." At the end

File: ashpd-demo/src/portals/desktop/location.rs [hunk 2]
```diff
@@ -63,6 +63,7 @@ mod imp {
     }
     impl ObjectImpl for LocationPage {
         fn constructed(&self, _obj: &Self::Type) {
+            // TODO Localize strings.
```
Comment (bilelmoussaoui) [inline ashpd-demo/src/portals/desktop/location.rs:66]: Don't for now, as we use from_str to re-get the original enums back

**Review by bilelmoussaoui — APPROVED** [review]
Other than the two comments I left, looks good to me!

File: ashpd-demo/src/portals/desktop/location.rs (hunk 2 above)
Comment (A6GibKm) [reply ashpd-demo/src/portals/desktop/location.rs]: This is not valid anymore.

**Comment by bilelmoussaoui** [comment]
Thanks!

**Pull request merged**